docs/guide/arktos_mizar_installation_single.md

```diff
@@ -0,0 +1,140 @@
+
+## Installation Steps
+1. Prepare lab machine, the prefered OS is **Ubuntu 18.04**. If you are using AWS, the recommanded instance size is ```t2.2xlarge``` and the storage size is ```128GB``` or more 
+
+```bash
+wget https://raw.githubusercontent.com/Hong-Chang/tools/master/setup-machine-arktos.sh && chmod 755 setup-machine-arktos.sh && ./setup-machine-arktos.sh
+
+```
+Once above script is completed, you will be automatically logged out of the lab machine. 
+
+2. You will need ssh into the same lab machine again, and download mizar source code onto your lab machine: 
+
+```bash
+cd /home/ubuntu/go/src/k8s.io/
+git clone https://github.com/centaurus-cloud/mizar.git
+cd mizar
+```
+
+3. Once Mizar source code is downloaded, you can then run ```bootstrap.sh``` script to bootstrap your lab machine. 
+
+4. Once bootstrap is completed, you can then compile mizar: 
+
+```bash
+cd /home/ubuntu/go/src/k8s.io/mizar
+make
+make proto
+```
+
+5. Deploy CNI and restart containerd: 
+```bash
+wget -qO- https://github.com/futurewei-cloud/containerd/releases/download/tenant-cni-args/containerd.zip | zcat > /tmp/containerd
+chmod +x /tmp/containerd
+sudo systemctl stop containerd
+sudo mv /usr/bin/containerd /usr/bin/containerd.bak
+sudo mv /tmp/containerd /usr/bin/
+sudo systemctl restart containerd
+sudo systemctl start docker
+```
+
+6. Build arktos-network-controller (as it is not part of arktos-up.sh yet)
+```bash
+make all WHAT=cmd/arktos-network-controller
+sudo rm -f /etc/resolv.conf
+sudo ln -s /run/systemd/resolve/resolv.conf /etc/resolv.conf
+```
+Also, please ensure the hostname and its ip address in /etc/hosts. For instance, if the hostname is ip-172-31-41-177, ip address is 172.31.41.177:
+```text
+127.0.0.1 localhost
+172.31.41.177 ip-172-31-41-177
+```
+
+7. Before installing Mizar, you will need first start up Arktos server: 
+
+```bash
+cd /home/ubuntu/go/src/k8s.io/arktos
+./hack/arktos-up.sh
+```
+
+Then wait till you see: 
+
+```
+To start using your cluster, you can open up another terminal/tab and run:
+
+  export KUBECONFIG=/var/run/kubernetes/admin.kubeconfig
+Or
+  export KUBECONFIG=/var/run/kubernetes/adminN(N=0,1,...).kubeconfig
+
+  cluster/kubectl.sh
+
+Alternatively, you can write to the default kubeconfig:
+
+  export KUBERNETES_PROVIDER=local
+
+  cluster/kubectl.sh config set-cluster local --server=https://ip-172-31-16-157:6443 --certificate-authority=/var/run/kubernetes/server-ca.crt
+  cluster/kubectl.sh config set-credentials myself --client-key=/var/run/kubernetes/client-admin.key --client-certificate=/var/run/kubernetes/client-admin.crt
+  cluster/kubectl.sh config set-context local --cluster=local --user=myself
+  cluster/kubectl.sh config use-context local
+  cluster/kubectl.sh
+```
+
+Once you see above text, your arktos server is now running. 
+To verify, you can run ```kubectl get nodes```, you should see a node running with the name starts with "ip" followed by the private ip address of your lab machine. 

```

Comment:
Need to use another ssh terminal. If don't mention it, user may be confused because the original terminal is stuck in arktos-up.